packages/dashboard/src/components/datagrid/DataGrid.tsx

```
@@ -105,16 +104,34 @@ export default function DataGrid<TRow extends DataGridRowType = DataGridRow>({
-  // Convert columns to react-data-grid format
+
+  const defaultColumnKeys = useMemo(
+    () => columns.map((c) => c.key),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [columns.map((c) => c.key).join(',')]
+  );
+
+  const { columnKeys, reorderColumns } = useColumnOrder(
+    storageKey ?? '',
+    defaultColumnKeys
+  );
```

ESLint

[error] Replace `⏎····storageKey·??·'',⏎····defaultColumnKeys⏎··` with `storageKey·??·'',·defaultColumnKeys`

(prettier/prettier)